integration/dataservice-hosting-tests/tests-integration/tests/src/test/java/org/wso2/ei/dataservice/integration/test/jira/issues/CARBON15262JsonGsonFormatterTenantModeTest.java

```diff
@@ -42,7 +42,7 @@
  *
  */
 public class CARBON15262JsonGsonFormatterTenantModeTest extends DSSIntegrationTest {
-	private final String serviceName = "H2SimpleJsonTest";
+	private final String serviceName = "GsonFormatterTenantModeTest";
 	private final String serviceName1 = "H2SimpleJsonTest1";
```

Comment:
Isn't the same service name used by another testcase? If it is used, we can change this also to avoid intermittent failures

Reply:
No It is not used anywhere else.